tests/resources/expectations/models.test_llama.TestLlama2GQA.test_model_output

```diff
@@ -1 +1 @@
-0.009829852730035782,0.033294279128313065,0.04122868552803993,0.013380158692598343,0.008465621620416641,0.015016693621873856,0.013299260288476944,0.022558819502592087,0.007371935993432999,0.033163610845804214,0.01397942379117012,0.008224960416555405,0.018899638205766678,0.016687992960214615,0.0,0.019778233021497726
\ No newline at end of file
+0.00982985645532608,0.03329431265592575,0.04122867435216904,0.01338011771440506,0.008465498685836792,0.015016578137874603,0.01329929381608963,0.022558867931365967,0.0073719024658203125,0.03316369652748108,0.013979442417621613,0.008224941790103912,0.01889961212873459,0.016687996685504913,0.0,0.019778311252593994
```

Comment:
the test_model_weights_keys of course needed to be regenerated due to changes to keys, but this one (and the other test_model_output below) is effectively the same (within some tolerance). does the re-generating script not allow for just regenerating the expects for the specific failed tests? or were these slight variations in the results actually out of tolerance for this test?